### PR TITLE
#165273591 Comment edit history

### PR DESCRIPTION
**Pull request by JamesMudidi**
#### What does this PR do?
- Sets up a way to view the history of the edited comment

#### Description of Task to be completed?
- When a comment has been edited, the user can go back and view all the history of the original version of the comment before it was edited

#### How should this be manually tested?
- When a comment has been edited, hit `http:127.0.0.1/api/articles/<article slug>/comments/<comment id>/history` endpoint to view all the edited comments

#### What are the relevant pivotal tracker stories?
- [165273591](https://www.pivotaltracker.com/story/show/165273591)

#### Screenshots (if appropriate)
Screenshot 2019-05-09 at 6 56 28 PM

### CODE REVIEW

File: authors/apps/comments/serializers.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from rest_framework import serializers
-from .models import (Comment,CommentLikeDislike)
+from .models import (Comment,CommentLikeDislike,CommentEditHistory)
```
Comment (bisonlou) [inline authors/apps/comments/serializers.py]: Please use pylint. If you do, there would be spaces after the commas.

**Review by bisonlou — CHANGES_REQUESTED** [review]
Great work. However, endeavour to use pylint.

**Review by nadralia — CHANGES_REQUESTED** [review]
NiceWork so far @JamesMudidi and @llwasampijja but let the comment history be attached to a specific comment. For now if i pass any id , it retrieve every comment history. you guys can also check out `[django-simple-history](https://pypi.org/project/django-simple-history/)` it's a library that you can use for history of models.

File: authors/apps/comments/models.py [hunk 2]
```diff
@@ -26,3 +27,10 @@ class CommentLikeDislike(models.Model):
 
     def __str__(self):
         return self.like
+
+class CommentEditHistory(models.Model):
+    "Model for collecting edited history of the comments"
```
Comment (kalsmic) [inline authors/apps/comments/models.py]: Also consider using triple quotes for docstrings.

**Review by ZackAtama — APPROVED** [review]
Good work @JamesMudidi  and @llwasampijja, am all okay with this PR soon as you've implemented requested changes from @bisonlou and @nadralia.

**Review by kalsmic — APPROVED** [review]
Well done @llwasampijja, now a comment history is attached to a specific comment. Also, the docstring format has been corrected.

**Review by nadralia — APPROVED** [review]

**Review by bisonlou — APPROVED** [review]

**Pull request merged**